src/components/dashboard/Dashboard.js

```diff
@@ -373,36 +380,43 @@ const Dashboard = props => {
           toValue: 68,
```

Comment:
i see the most of the animation options are the same, so this could be replaced onto .map() between array of 'toValue' props' values

Reply:
its not part of thiis PR, and we already discussed the conditions for a meaningful refactoring